chemprop/models/model.py

```diff
@@ -49,7 +49,16 @@ def create_encoder(self, args: TrainArgs) -> None:
         :param args: A :class:`~chemprop.args.TrainArgs` object containing model arguments.
         """
         self.encoder = MPN(args)
-
+              
+        if hasattr(args,'checkpoint_frzn'):
+            if (args.frzn_mpnn) | (args.frzn_ffn_layers>0):
```

Comment:
If you change the first layer of logic check to be a check for None, then you don't need this or statement. You can make this just the greater than zero check.

Stylistically, other places in chemprop where we have "or" operations it's been done with the word "or" instead of "|" so if there was a need to keep the or, then I would recommend changing the style.

Reply:
If the user freezes ffn layers, but not the mpnn, right now the function in utils.py prints a warning that the mpnn will also be frozen and overwrites the mpnn and ffn layers. Because of this if the user freezes ffn layers but not the mpnn, I want this to also trigger freezing the mpnn. Let me know if this doesn't make sense or if I'm wrong (I'm changing the or statements).